ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,36 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# instance_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.instance_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    eps=st.floats(min_value=0.01, max_value=0.1),
+)
+def test_paddle_instance_norm(
+    *,
+    values_tuple,
+    normalized_shape,
+    eps,
+    test_flags,
+    frontend,
+    on_device,
+    fn_tree,
+):
+    (dtype, x, normalized_shape, scale, offset) = values_tuple
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        on_device=on_device,
+        fn_tree=fn_tree,
+        x=x[0],
+        normalized_shape=normalized_shape,
```

Comment:
We should be testing for running_mean, running_var, use_input_stats, momentum and data format. All parameters of `paddle.nn.functional.instance_norm` must be included here. Also, there is no `normalized_shape` in `instance_norm`.